spec/models/manageiq/providers/hawkular/middleware_manager_spec.rb

```diff
@@ -159,7 +159,7 @@ def timeline_domain_expectations(status)
           ems.public_send("#{operation}_middleware_domain_server",
                           mw_domain_server.ems_ref.sub(/%2Fserver%3D/, '%2Fserver-config%3D'),
```

Comment:
@josejulio, I think you need to check these specs. They are playing with `ems_ref` like they were canonical paths. I guess that now it's not meant to be like this.